EasyPost/Client.cs

```diff
@@ -20,6 +21,7 @@ public class Client {
             configuration = clientConfiguration;
 
             client = new RestClient(clientConfiguration.ApiBase);
+            Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
```

Comment:
This seems like it would clobber an existing `CurrentCulture` set by a user of this client. My thought would be that this should be left up to the user, not the library.